src/components/navbar/header.js

```diff
@@ -25,48 +51,118 @@ const theme = createTheme({
       height: "90px",
       marginTop: "7px",
     },
+    menuIconStyling: {
+      color: WHITE_SOLID,
+      fontSize: "2.5rem",
+    },
+    slideIconLeftStyle: {
+      width: "50px",
+      height: "50px",
+      position: "absolute",
+      right: "30px",
+      top: "120px",
+      zIndex: "100",
+    },
   },
 });
+
 export default function Navbar() {
+  const theTheme = useTheme();
+  const [open, setOpen] = useState(false);
+  const [slideLeftIconVisiable, setSlideLeftIconVisiable] = useState(true);
+  const screenWidthIsLessThan600px = useMediaQuery(
+    theTheme.breakpoints.down("sm")
+  );
+  const screenWidthIsGreaterThan600px = useMediaQuery(
+    theTheme.breakpoints.up("sm")
+  );
+  function closeAndOpenSlideLeftIcon() {
+    if (screenWidthIsGreaterThan600px) setSlideLeftIconVisiable(true);
+    else if (screenWidthIsLessThan600px) {
+      setTimeout(function () {
+        return setSlideLeftIconVisiable(false);
+      }, 6000);
+    }
+  }
+  useEffect(() => {
+    closeAndOpenSlideLeftIcon();
+  });
+
+  const classes = useStyles();
   return (
     <Box sx={{ flexGrow: 1 }}>
       <ThemeProvider theme={theme}>
-        <AppBar style={theme.custom.header}>
-          <Toolbar>
-            <Typography sx={{ flexGrow: 1 }}>
+        <AppBar position="sticky" style={theme.custom.header}>
```

Comment:
why are you adding `position: "sticky",` here? 
there's `position: "static",` in `theme.custom.header`